test/data/nomisClientTest.js

```diff
@@ -535,20 +552,40 @@ describe('nomisClient', () => {
         .put('/offender-sentences/booking/aaa/home-detention-curfews/latest/checks-passed')
         .reply(200, { result: 'answer' })
 
-      return expect(nomisClient.putChecksPassed('aaa')).to.eventually.eql({ result: 'answer' })
+      return expect(nomisClient.putChecksPassed({ bookingId: 'aaa', passed: true })).to.eventually.eql({
+        result: 'answer',
+      })
     })
 
-    it('should pass in passed=true  and date', () => {
+    it('should pass in passed value and date', () => {
       fakeNomis
         .put('/offender-sentences/booking/aaa/home-detention-curfews/latest/checks-passed', {
-          passed: 'true',
+          passed: true,
           date: '2018-05-31',
         })
         .reply(200, { result: 'answer' })
 
-      return expect(nomisClient.putChecksPassed('aaa', { approvalStatus: 'status' })).to.eventually.eql({
+      return expect(nomisClient.putChecksPassed({ bookingId: 'aaa', passed: true })).to.eventually.eql({
         result: 'answer',
       })
     })
+
+    it('should throw error if passed is undefined', () => {
```

Comment:
I think this can be done with 2 tests
```
it('should throw error if passed is undefined', () => {
```
and

```
it('should throw error if passed is not a boolean', () => {
```

I don't think it's necessary checking for the other types?

Reply:
Arguably you probably only need 
```
it('should throw error if passed is not a boolean', () => {
```
What do you think?

Reply:
Agreed. I was trying to make it explicit and clear, but "not boolean" is pretty clear.